cloud/docker/docker.py

```diff
@@ -725,16 +725,18 @@ def has_changed(self):
         return False
 
     def get_inspect_image(self):
+        return self.client.inspect_image(self.module.params.get('image'))
+
+    def get_image_repo_tags(self):
```

Comment:
I'm a bit curious about the `get_inspect_image` / `get_image_repo_tags` split here. Did this solve a specific problem you encountered, or is it just a better client method to call?

Also, does `self.client.inspect_image` still work if you leave off the image tag?